### ISSUE TITLE
About finetune steps when finetuning Openvla on Bridge Data V2

### ISSUE THREAD

**Issue by zzj1111** [openvla/openvla#73]
I finetuned Openvla on Bridge Data V2 with batchsize=4 and grad_accumulation_steps=2, but I found that the num of the total steps is 200000, which took 72 A100 hours. Does it normal?

**Comment by kpertsch** [comment]
As mentioned in the other issue, finetuning on BridgeDataV2 isn't very meaningful since OpenVLA already performs very well on BridgeData out of the box (95%+ action token accuracy). So it would be better to collect your own data or use a dataset that is not in the pre-training mix.

As an aside: the Bridge dataset is quite large and diverse (60k episodes), so a batch size as small as 8 may be too small to get a sufficiently noise-free gradient signal for convergence. You may want to increase the batch size if your GPUs can fit it!
